docs/index.md

```diff
@@ -56,7 +56,7 @@ redirect_from: "/start"
     <div class="col-md-6 feature">
       <img src="/img/feature-icons/integrations-icon.svg" alt="Integrations icon">
       <h2>Integrations</h2>
-      CyberArk officially provides and supports integration libraries between Conjur and external tools such as Puppet, Ansible, Chef, Jenkins, Salt Stack, Docker, Kubernetes, OpenShift, and CloudFoundry. CyberArk has officially partnered with Puppet to provide joint support for the Conjur Puppet Module. CyberArk is extending this partnering relationship to other major tool vendors in the DevOps space.
+      CyberArk officially provides and supports integration libraries between Conjur and external tools such as Puppet, Ansible, Chef, Jenkins, Salt Stack, Docker, and Kubernetes. CyberArk has officially partnered with Puppet to provide joint support for the Conjur Puppet Module. CyberArk is extending this partnering relationship to other major tool vendors in the DevOps space.
```

Comment:
Remove Chef, Jenkins, Salt Stack, Docker (maybe?), and Kubernetes. I'm not sure what an integration library with Docker really means. We're certainly using it heavily.